src/Bitcoin/InputSelector.cpp

```diff
@@ -74,7 +74,7 @@ InputSelector<TypeWithAmount>::select(int64_t targetValue, int64_t byteFee, int6
     }
 
     // total values of utxos should be greater than targetValue
-    if (inputs.empty() || sum(inputs) < targetValue) {
+    if (inputs.empty() || sum(inputs) < static_cast<uint64_t>(targetValue)) {
```

Comment:
Please change `targetValue` and all parameters of `select()` to `uint64_t`, so this case is not needed here. This applies to all `int64_t` parameters in `InputSelector.h`.